services/director-v2/src/simcore_service_director_v2/modules/comp_scheduler/base_scheduler.py

```diff
@@ -250,17 +250,19 @@ async def _schedule_pipeline(
             )
 
             # find the tasks that need scheduling
-            tasks_to_schedule = [node_id for node_id, degree in pipeline_dag.in_degree() if degree == 0]  # type: ignore
+            iter_tasks_to_schedule: Iterable[NodeID] = (
+                node_id for node_id, degree in pipeline_dag.in_degree() if degree == 0
```

Comment:
good one... but I'm going to change that file. Could you actually revert it? no need to add conflicts with my PR